Makefile

```diff
@@ -15,8 +15,28 @@ SRCFILES := *.go
 all: firectl
 
 firectl: $(SRCFILES)
-	go build
+	go build -o firectl
 
+release: $(SRCFILES)
+	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build \
+		    -a \
+		    -installsuffix cgo \
+		    -ldflags "-s" \
+		    -gcflags=all=-trimpath=${TRIMPATH} \
+		    -asmflags=all=-trimpath=${TRIMPATH} \
```

Comment:
Can you give some context why is this needed?

Reply:
Ah, basically if you don't include that you'll have your machines paths on there. For example, if you do `strings firectl | grep /home/mxpv` it'd spit out a bunch of paths. So to generate binaries with no knowledge of what host it is being built on, you have to strip them during compilation.

Reply:
What we've done in other projects (like the ECS agent) is to define our release process such that builds always happen inside a particular, well-defined Docker container with a predictable path that we don't particularly care about.  I don't think trimpath is strictly necessary if we do that, and I would like us to use a well-defined Docker container for our builds so we can have as much of a reproducible environment as possible.